Remote sessions guide documents a client option that does not exist

## Summary

`docs/features/remote-sessions.md` tells users to enable Mission Control remote sessions by setting a client option named `remote` (TypeScript) / `Remote` (Go, C#). That option does not exist in those bindings. The real one is `enableRemoteSessions` (TypeScript) / `EnableRemoteSessions` (Go, C#), so copy-pasting the documented client construction fails to compile.

The Python and Rust tabs in the same section already use the correct name, so the guide currently contradicts itself within one section.

## Reproduction (TypeScript, published package)

```bash
mkdir repro && cd repro
printf '{ "name":"repro","private":true,"type":"module","version":"0.0.0" }' > package.json
npm install @github/copilot-sdk typescript
```

`tsconfig.json`:

```json
{
  "compilerOptions": {
    "target": "ES2022", "module": "NodeNext", "moduleResolution": "NodeNext",
    "strict": true, "noEmit": true, "skipLibCheck": true
  },
  "files": ["broken.ts"]
}
```

`broken.ts`, containing the guide's client-construction line verbatim:

```ts
import { CopilotClient } from "@github/copilot-sdk";

const client = new CopilotClient({ remote: true });
void client;
```

Result:

```
$ npx tsc --noEmit
broken.ts(3,36): error TS2353: Object literal may only specify known properties, and 'remote' does not exist in type 'CopilotClientOptions'.
```

Reproduced with `@github/copilot-sdk@1.0.8` and TypeScript 7.0.2. Substituting `enableRemoteSessions: true` compiles cleanly (exit 0).

## Same defect in the Go and C# examples

Verified against `main`, each compiled against a local build of the affected binding:

| Tab | Documented | Declared in the SDK | Compiler error |
| --- | --- | --- | --- |
| TypeScript | `new CopilotClient({ remote: true })` | `enableRemoteSessions?: boolean` (`nodejs/src/types.ts`) | `TS2353 ... 'remote' does not exist in type 'CopilotClientOptions'` |
| Go | `copilot.NewClient(&copilot.ClientOptions{Remote: true})` | `EnableRemoteSessions bool` (`go/types.go`) | `unknown field Remote in struct literal of type copilot.ClientOptions` |
| C# | `new CopilotClientOptions { Remote = true }` | `public bool EnableRemoteSessions { get; set; }` (`dotnet/src/Types.cs`) | `CS0117: 'CopilotClientOptions' does not contain a definition for 'Remote'` |

The Go example additionally has an independent error on the same line: it writes `client, _ := copilot.NewClient(...)`, but `NewClient` returns a single value, so `go build` reports both errors together.

## Additional stale text in the same guide

The bullet under `## Notes` names the same nonexistent option and states that it "only applies when the SDK spawns the CLI process". That is also inaccurate: the option is honored both when the SDK starts the runtime as a child process and when it starts an in-process host. It is ignored only when connecting to an already-running runtime.

## Root cause

The client option was renamed for cross-SDK consistency during the per-language API reviews: C# in #1343, TypeScript in #1357, and Go in #1360. None of those touched this guide. The Rust (#1367) and Python (#1376) reviews did update their own tabs in it, which is why only those two tabs are correct today.

Every fenced block in this file carries `<!-- docs-validate: skip -->`, so the documentation validation never compiled these snippets and the drift went unnoticed.

--- a/docs/features/remote-sessions.md
+++ b/docs/features/remote-sessions.md
@@ -13,7 +13,7 @@ For running sessions on GitHub-hosted compute, see [Cloud Sessions](./cloud-sess
 
 ### Always-on (client-level)
 
-Set `remote: true` when creating the client. Every session in a GitHub repo automatically gets a remote URL.
+Set `enableRemoteSessions: true` when creating the client. Every session in a GitHub repo automatically gets a remote URL.
 
 <!-- tabs:start -->
 
@@ -23,7 +23,7 @@ Set `remote: true` when creating the client. Every session in a GitHub repo auto
 ```typescript
 import { CopilotClient } from "@github/copilot-sdk";
 
-const client = new CopilotClient({ remote: true });
+const client = new CopilotClient({ enableRemoteSessions: true });
 const session = await client.createSession({
   workingDirectory: "/path/to/github-repo",
   onPermissionRequest: async () => ({ allowed: true }),
@@ -59,7 +59,7 @@ session.on(on_event)
 
 <!-- docs-validate: skip -->
 ```go
-client, _ := copilot.NewClient(&copilot.ClientOptions{Remote: true})
+client := copilot.NewClient(&copilot.ClientOptions{EnableRemoteSessions: true})
 session, _ := client.CreateSession(ctx, &copilot.SessionConfig{
     WorkingDirectory: "/path/to/github-repo",
     OnPermissionRequest: func(req copilot.PermissionRequest, inv copilot.PermissionInvocation) (rpc.PermissionDecision, error) {
@@ -78,7 +78,7 @@ session.On(func(event copilot.SessionEvent) {
 
 <!-- docs-validate: skip -->
 ```csharp
-var client = new CopilotClient(new CopilotClientOptions { Remote = true });
+var client = new CopilotClient(new CopilotClientOptions { EnableRemoteSessions = true });
 var session = await client.CreateSessionAsync(new SessionConfig
 {
     WorkingDirectory = "/path/to/github-repo",
@@ -201,6 +201,6 @@ The remote URL can be rendered as a QR code for easy mobile access. The SDK prov
 
 ## Notes
 
-* The `remote` client option only applies when the SDK spawns the CLI process. It is ignored when connecting to an external server via `cliUrl`.
+* The `enableRemoteSessions` client option applies when the SDK starts the runtime, either as a child process or as an in-process host. It is ignored when connecting to an already-running runtime.
 * If the working directory is not a GitHub repository, remote setup is silently skipped (always-on mode) or returns an error (on-demand mode).
 * Remote sessions require authentication. Ensure `gitHubToken` or `useLoggedInUser` is configured.

